### ISSUE TITLE
Update IA Guidance: Breadcrumbs

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#73451]
Problem: implementation of form H1s, breadcrumbs, and title tags is all over the place. Teams struggle to understand what they should do, and CAIA struggles to understand what guidance to use as standards change over time. 

We should vet the current documentation on this in VADS and see what changes we could make to improve it.

**Comment by kristinoletmuskat** [comment]
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/df62e5e7-1a08-4cd1-bf8c-603e95f758c4)
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/a1adf0a2-9b3b-4f68-9f1a-a9e8ccb88a1e)
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/55166c42-aece-4a85-8780-649da19d4e6b)

**Comment by kristinoletmuskat** [comment]
Here's another one -- the new header thing doesn't include breadcrumb guidance for desktop.

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/a165bfd3-d191-4892-86aa-0861e4a32f7f)

**Comment by coforma-jamie** [comment]
The CAIA accessibility specialists met to discuss `<title>` options.

## Recommendation
CAIA advises `Form Title | Veterans Affairs` for these reasons:

### SEO
You want unique page <title>s so that search results have meaning. a search result “What’s your name? | Veterans Affairs” wouldn’t be meaningful; “Apply for aid | Veterans Affairs” would be

### Keyboard users navigating between multiple windows/tabs
Keyboard users often tab between browser tabs or open browser windows. On windows you use Alt + Tab to do that. When you do, you get thumbnails of the different windows and, if it’s a browser window we’re talking about, THE TITLE TAG’s text.
So imagine you’re filling out 2 or 3 forms at a clip, they’re each in their own window, and you’re alt + tabbing between them. Each could be “What’s your name? | Veterans Affairs” — you wouldn’t be able to tell them apart.

**Comment by coforma-jamie** [comment]
I'll review the team's Mural for accessibility guidance once it's been created. @kristinoletmuskat, when the Mural is ready for review, please re-assign me to this ticket, and tag me. :)

**Comment by coforma-jamie** [comment]
Form `<title>` should be `Form Title | Veterans Affairs`

We (@kristinoletmuskat, @laflannery, @jeana-adhoc, and I) decided on this because:

- Duplicate titles are a usability issue - whether they are a WCAG violation is debatable;
- Our step pattern makes generating unique <titles> near impossible;
- It’s unlikely that users will have more than one step of a form open at the same time; and
- Some teams send focus to the form question on page load. This helps clear up confusion for screen readers that may arise from duplicate <title> tags.

**Comment by kristinoletmuskat** [comment]
FYI I'm going to change this ticket to be specifically about breadcrumbs since we have title tags in another one and H1s we don't really need to add much I don't think! 

Did some drafting work on breadcrumb standards and exceptions if anyone wants to check it out. I want to review a second time with Sara before sending to Mikki.

https://docs.google.com/document/d/1XKUSwFIaqVYq_ZuXAoZ33sWjDWgTFWdZVMYJescmZGA/edit

**Comment by coforma-jamie** [comment]
I wrote a couple of comments / questions for you in the Google Doc, @kristinoletmuskat

**Comment by kristinoletmuskat** [comment]
Adding this ticket Mikki just created: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2950

**Comment by kristinoletmuskat** [comment]
Hey hey just an update on this: 

- we've published the updated breadcrumb guidance for forms (ie, 'freeze' the breadcrumb at the form title, which when clicked should return user to first page of the form) and it's live on [VADS](https://design.va.gov/components/breadcrumbs). 
- But, we're in the process of potentially moving to a different solution that leverages <Back, which might make more sense for some of the authenticated forms and use cases. So I'm going to keep this open for now and actually tag @sterkenburgsara on this, since we're working on that guidance together. There's some coordiantion with the DST that needs to happen, so this ticket will likely be open for awhile. A11y folks feel free to untag yourselves unless you want to stay in the loop on this work moving forward!

**Comment by kristinoletmuskat** [comment]
Latest draft sent to Mikki and IA team for review: https://docs.google.com/document/d/1XKUSwFIaqVYq_ZuXAoZ33sWjDWgTFWdZVMYJescmZGA/edit?tab=t.0

**Comment by mnorthuis** [comment]
Reviewed and provided feedback.

**Comment by kristinoletmuskat** [comment]
@erinrwhite @mnorthuis I think we are ready to publish!

**Comment by mnorthuis** [comment]
@kristinoletmuskat I reviewed the breadcrumbs and made some modifications.  With those changes, this is ready to go.  Let me know if there's any questions or if I made any mistakes.

**Comment by erinrwhite** [comment]
@kristinoletmuskat confirming [this google doc](https://docs.google.com/document/d/1XKUSwFIaqVYq_ZuXAoZ33sWjDWgTFWdZVMYJescmZGA/edit?tab=t.0#heading=h.efzncwe2ayez) is our source of truth for the new guidance?

**Comment by kristinoletmuskat** [comment]
yes!

**Comment by erinrwhite** [comment]
[PR submitted!](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/3614)

[Preview how these changes will look](https://dev-design.va.gov/3614/components/breadcrumbs)

**Comment by mnorthuis** [comment]
Updates published and [announced in slack](https://dsva.slack.com/archives/C0NGDDXME/p1734535748861849)
